app/helpers/service_provider_helper.rb

```diff
@@ -88,6 +88,11 @@ def sp_allow_prompt_login_img_alt(sp_allows_prompt_login)
     'prompt=login disabled'
   end
 
+  def sp_email_nameid_format_allowed_img_alt(sp_email_nameid_format_allowed)
```

Comment:
All the view stuff is getting very repetitive; I have another JIRA ticket for allowing admins to manage another attribute (the signed response one) so I think I'll clean up / refactor in that one.